config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/mechanics/:id", to: "mechanics#show"
+  post "/mechanics/:id/rides", to: "mechanics#add_ride"
```

Comment:
This is very close but instead of pointing to the `mechanics` controller and a `add_ride` action, we would want it to point to `ride_mechanics#create`.